client/src/helpers/APICalls/profile.ts

```diff
@@ -0,0 +1,16 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const createProfile = async (): Promise<AuthApiData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    credentials: 'include',
```

Comment:
If we intend to create a profile using this helper function, we would also need to pass the relevant input parameters in the request body here!